.ado/publish.yml

```diff
@@ -182,14 +188,18 @@ jobs:
             React.Windows.Desktop.DLL\**
             React.Windows.Desktop.Test.DLL\**
       
+      - script: |
+          echo ##vso[task.setvariable variable=EnableCodesign]true
+        displayName: Enable Codesign
+        condition: or(${{ parameters.enableCodesign }}, endsWith(variables['Build.SourceBranchName'], '-stable'))
```

Comment:
Since there's two inputs to enable code signing, it might make sense to rename the "enable code signing" param to instead be negative.

I.e. the value of the param can't guarantee code signing is enabled, but it can disable it.

Reply:
I can try to make the parameter clearer, I just want to make sure:

1. We *never* codesign if CG has alerts. There should be no easy mechanism for bypassing this.
2. Codesign is enabled by default on -stable branches
3. Codesign is disabled by default on master